src/app/state/rest/config.tsx

```diff
@@ -11,27 +12,23 @@ type APIListResponse<T> = {
 
 export const useCases = () =>
   useApiRequest<APIListResponse<API.Case>>({
-    url: "cases",
-    group: "cases" // <- When a POST/PUT/PATCH/DELETE (anything other then GET) is called, cache for the whole group is invalidated
+    url: slashSandwich([process.env.REACT_APP_GATEWAY, "cases"])
```

Comment:
Decided to pass the full URL here.
We can use `useApiRequest` also to query other APIs (such as BAG)

Reply:
Eventually it might be nice to have all hosts and endpoints configured in one JSON file / TypeScript type

Reply:
Yeah I like that.